src/main/java/org/jboss/processing/NewIssueCollector.java

```diff
@@ -38,9 +39,9 @@ public static NewIssueCollector getInstance(JiraEndpoint jiraEndpoint) {
     }
 
     @Override
-    public EveryIssueState execute() throws Exception {
+    public void execute(LotteryConfig lotteryConfig) throws Exception {
         int issuesCount = doPoll();
         Log.infof("Found number of issues %d", issuesCount);
-        return new EveryIssueState(state.issueStates);
+        new EveryIssueState(state.issueStates);
```

Comment:
why are you creating object that is immediately going to be collected? Do you want to do something with this instance?